libparsec/crates/platform_mountpoint/src/lib.rs

```diff
@@ -17,6 +17,11 @@ pub(crate) use unix as platform;
 #[cfg(target_os = "windows")]
 pub(crate) use windows as platform;
 
+#[cfg(target_os = "windows")]
+// TODO: remove me once winify is used
+#[allow(unused_imports)]
+pub(crate) use windows::{unwinify_entry_name, winify_entry_name};
+
```

Comment:
**chore**: You don't use it, remove it

Reply:
Done

Reply:
It's exported in `libparsec/crates/platform_mountpoint/src/windows/mod.rs` you can use them in `libparsec/crates/platform_mountpoint/tests/unit/winify.rs` that would allow to remove it.